lib/insights.ts

```diff
@@ -32,7 +32,12 @@ import {
   InsightsSearchViewFiltersEvent,
   viewedFilters
 } from "./view";
-import { ANONYMOUS_USER_TOKEN, getUserToken, setUserToken } from "./_cookieUtils";
+import {
+  ANONYMOUS_USER_TOKEN,
+  getUserToken,
+  setUserToken
+} from "./_cookieUtils";
+import { version } from "../package.json";
```

Comment:
Be careful to not embed the full `package.json` inside the lib.

Reply:
yes, I was worried about that at first, but rollup-plugin-json is smart about it and even inlines constants